ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/ActiveContractsService.scala

```diff
@@ -300,6 +300,24 @@ class ActiveContractsService(session: LedgerSession) extends LedgerTestSuite(ses
       }
     }
 
+  private val eventId =
+    LedgerTest("ACSeventId", "The ActiveContractService should properly fill the eventId field") {
```

Comment:
Good question. It would not have failed in the sandbox, but only when running the test with the reference-server v2 (using inmemory kvutils participant), where contract ids actual aren't the same as event ids.
I've ran this test without the change and it failed as expected.